test/upgrade/prober/wathola/receiver/services.go

```diff
@@ -69,6 +69,8 @@ func (r receiver) receiveEvent(ctx context.Context, e cloudevents.Event) {
 		err := e.DataAs(step)
 		if err != nil {
 			log.Fatal(err)
+		} else {
```

Comment:
The "else" branch probably doesn't have to be here, log.Fatal ensures the code will stop executing.
It could be simplified to:
```
if err != nil {
  log.Fatal(err)
}
span.AddAttributes(trace.Int64Attribute("step", int64(step.Number)))
r.step.RegisterStep(step)

```

Reply:
But it's a minor thing.